### ISSUE TITLE
Project level changeset streaming with OSMCha

### ISSUE THREAD

**Issue by xamanu** [hotosm/tasking-manager#1379]
Often project managers don't have a good sense of what's happening in their data. Create a link to OSMCha with all the boundaries of the project, the time range since the project started, and all contributors who mapped in this project. This way admins or PMs could easily look at the changesets and see what is wrong, and who is doing poorly.

**Comment by xamanu** [comment]
cc: @russdeffner, @nrotstan

**Comment by russdeffner** [comment]
Yes, this seems like a good thing. Maybe even open it up to validators as well?

**Comment by ramyaragupathy** [comment]
Existing feature to add a `OSMCha filter ID` seems to link to the filter directly. Were there other considerations for automating this implementation?
